resolwe_bio/processes/alignment/bowtie.yml

```diff
@@ -358,6 +358,12 @@
           description: >
             searches for at most X distinct, valid alignments for each read. The search terminates when it can't find more distinct valid alignments, or when it finds X, whichever happens first.          default: 5
           default: 5
+    - name: threads
```

Comment:
Yes, this should work if the given requirement is set.

Note that I remember a discussion about moving this `requirements` into a different namespace due to including `contributor` as well. In that case, this will have to be updated as well.